Sprint-3/slideshow/slideshow.js

```diff
@@ -1,8 +1,63 @@
 const images = [
-    "./assets/cute-cat-a.png",
-    "./assets/cute-cat-b.jpg",
-    "./assets/cute-cat-c.jpg",
+  "./assets/cute-cat-a.png",
+  "./assets/cute-cat-b.jpg",
+  "./assets/cute-cat-c.jpg",
 ];
 
+// Write your code here
 
-// Write your code here
\ No newline at end of file
+let currentIndex = 0;
+const carouselImg = document.getElementById("carousel-img");
```

Comment:
In general your naming in this PR is really good. The variables are really clear what they are and what they're for, the functions have really clear names - I can tell what they do from just their names, etc. Good job!